Makefile

```diff
@@ -505,6 +524,30 @@ $(BUILD_DIR)/assets/%.jpg.inc.c: assets/%.jpg
 $(BUILD_DIR)/%.schl.inc: %.schl
 	$(SCHC) $(SCHC_FLAGS) -o $@ $<
 
+# Audio
+
+AUDIO_BUILD_DEBUG ?= 0
```

Comment:
I wouldn't mind moving this to the top of the Makefile with the other settings we have, but since this is kinda niche I guess is fine to have it here

Reply:
I think I'd like to leave it here for now at least, it can always be moved later